### PR TITLE
Wd

### LINKED ISSUE #39: separate passing and failing fastqs/fast5
End users would like to keep the fastqs/fast5s separated into their passing/failing categories.

### LINKED ISSUE #43: refactor pod5 conversion
Currently, the pod5 conversion takes place in a function called `read_flowcell_info`. It would be good to move it an appropriate function or the snakemake pipeline.

### LINKED ISSUE #44: fastq/ directory duplication
Currently the `fastq/` directory contains a duplicate of what is in the project directory. One or the other should be chosen. 

If we are keeping the fast5 or pod5 then it should be safe to get rid of this.

### PR DESCRIPTION
**Pull request by WardDeb**
WIP regarding some enhancements:

- closes #44 
- closes #43 
- closes #39 
    - [x] rename for no_bc runs
    - [x] rename for bc runs
    - [x] analysis only on passing, no_bc runs
    - [x] analysis only on passing, bc runs

### CODE REVIEW

**Comment by WardDeb** [comment]
tested now on 2 non-bc, 2 bc flow cells.

File: src/npr/ont.py [hunk 1]
```diff
@@ -105,27 +113,42 @@ def sleep():
...
+            conda_prefix = conda_prefix if conda_prefix else None
+
+            print("Starting snakemake on file {} with configfile {} using workdir {}..."
+                  .format(snakefile_file, configFile, output_directory), file=sys.stderr)
             snak_stat = snakemake.snakemake(
                 snakefile = snakefile_file,
-                debug = True,
-                #cores = 25,
+                #debug = True,
+                cores = config["snakemake"]["cores"],
```
Comment (dkoppstein) [inline src/npr/ont.py:125]: One option here might be to simply pass **config['snakemake'] to snakemake.snakemake. This way, any arbitrary snakemake options can be passed via the config.

File: src/npr/communication.py [hunk 2]
```diff
@@ -8,28 +8,39 @@
...
     )
     mailer['From'] = config['email']['from']
-    mailer['To'] = config['email']['to']
+    if allreceivers:
+        mailer['To'] = config['email']['to']
+        tomailers = config['email']['to'].split(',')
+    else:
+        mailer['To'] = config['email']['trigger']
+        tomailers = config['email']['trigger'].split(',')
+    print("Email trigger, sending to {}".format(tomailers))
```
Comment (dkoppstein) [inline src/npr/communication.py:24]: alternatively: 
```
to_email = 'to' if allreceivers else 'trigger'
mailer['To'] = config['email'][to_email]
tomailers = mailer['To'].split(,')
print("Email trigger, sending to {}".format(tomailers))
```

**Review by dkoppstein — APPROVED** [review]
Looks good. I think 2_renamed.smk can be simplified significantly, which will also make maintaining it easier, but I will add a separate issue for this.

**Pull request merged**